src/peakrdl_sv/node.py

```diff
@@ -9,8 +9,14 @@
 
 
 class Node(UserList):
-    def __init__(self, node: AddrmapNode | RegfileNode | RegNode | FieldNode) -> None:
+    def __init__(
+        self,
+        node: AddrmapNode | RegfileNode | RegNode | FieldNode,
+        parent: AddrmapNode | RegfileNode | RegNode | None,
```

Comment:
The `parent` should be one of our wrapped classes, not the base RDL class.

Reply:
Fixed.